### DISCUSSION TITLE
How do I use IPF coloring with -slicemesh visualization

### DISCUSSION THREAD

**Discussion by kkp5295** [neperfepx/neper#891] [Q&A]
I have a mesh file and would like to color a slice of my microstructure using ipf coloring but keep getting segmentation faults whenever I put in -dataeltcolscheme ipf.

For example, this works fine but does not color by ipf:
neper -V simulation.msh -dataeltcol ori  -slicemesh z=0.1 -cameracoo 0:0:4 -print pic

But this gives segmentation fault:
neper -V simulation.msh -dataeltcol ori  -dataeltcolscheme ipf -slicemesh z=0.1 -cameracoo 0:0:4 -print ipfpic

How should I place the color scheme arguments?

**Comment by rquey** [comment]
Make sure to define the crystal symmetry. Here is an example:
```
neper -T -n 10 -reg 1 -crysym cubic -o simulation
neper -M simulation.tess
neper -V simulation.msh -dataeltcol ori -slicemesh z=0.1 -cameracoo 0:0:4 -camerasky 0:1:0 -print pic
```
